app/assets/src/components/views/phylo_tree/PhyloTreeByPathCreation.jsx

```diff
@@ -349,91 +432,159 @@ class PhyloTreeByPathCreation extends React.Component {
+          key="wizard__page_4"
+          title={`Would you like additional samples from IDSeq that contain ${
+            this.state.taxonName
+          }?`}
+        >
+          <div className="wizard__page-4__subtitle" />
+          <div className="wizard__page-4__searchbar">
+            <div className="wizard__page-4__searchbar__container">
+              <Input placeholder="Search" onChange={this.handleFilterChange} />
+            </div>
+            <div className="wizard__page-4__searchbar__container">
+              {this.state.selectedProjectSamples.size} Project Samples
+            </div>
+            <div className="wizard__page-4__searchbar__container">
+              {this.state.selectedOtherSamples.size} IDSeq Samples
+            </div>
+            <div className="wizard__page-4__searchbar__container">
+              {this.getTotalPageRendering()}
+            </div>
           </div>
-          <div className="wizard__page-3__searchbar__container">
-            {this.getTotalPageRendering()}
+          <div className="wizard__page-4__table">
+            {this.state.samplesLoaded && this.state.otherSamples.length > 0 ? (
+              <DataTable
+                headers={this.otherSamplesHeaders}
+                columns={otherSamplesColumns}
+                data={this.state.otherSamples}
+                selectedRows={this.state.selectedOtherSamples}
+                onSelectedRowsChanged={this.handleChangedOtherSamples}
+                filter={this.state.otherSamplesFilter}
+              />
+            ) : this.state.samplesLoaded &&
+            this.state.otherSamples.length == 0 ? (
+              <div>No samples containing {this.state.taxonName} available</div>
+            ) : (
+              <LoadingSpinner />
+            )}
           </div>
-        </div>
-        <div className="wizard__page-3__table">
-          {this.state.samplesLoaded && (
-            <DataTable
-              headers={this.otherSamplesHeaders}
-              columns={otherSamplesColumns}
-              data={this.state.otherSamples}
-              selectedRows={this.state.selectedOtherSamples}
-              onSelectedRowsChanged={this.handleChangedOtherSamples}
-              filter={this.state.otherSamplesFilter}
-            />
-          )}
-        </div>
-      </Wizard.Page>
+        </Wizard.Page>
+      )
+    };
+    return options[action];
+  }
+
+  getPages() {
+    let chosenPages = [];
+    if (!this.state.skipListTrees) {
+      chosenPages.push(this.page("listTrees"));
+    }
+    if (!this.skipSelectProjectAndTaxon) {
+      chosenPages.push(this.page("selectTaxonAndProject"));
+    }
+    chosenPages.push(
+      this.page("selectNameAndProjectSamples"),
+      this.page("addIdseqSamples")
     );
-    return pages;
+    return chosenPages;
   }
 
   render() {
     if (this.state.phyloTreesLoaded) {
+      let nPagesToSkip =
+        (this.state.skipListTrees | 0) + (this.skipSelectProjectAndTaxon | 0);
```

Comment:
I would add page info for the project and taxon information. I consider that part of creating the tree.
You would have 4 pages instead of 2.

Reply:
ok fixed